src/components/asset-list/RecyclerAssetList.js

```diff
@@ -633,7 +633,7 @@ class RecyclerAssetList extends Component {
     }
 
     if (row.item && row.item.smallBalancesContainer) {
-      return `balance_${row.item.stableId}`;
+      return `smallBalancesContainer`;
```

Comment:
there's no stableId on the "smallBalancesContainer" so it was just doing balance_undefined